index.html

```diff
@@ -1,13 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="style.css"> 
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
 <body>
-    <h1>Hola Mundo</h1>
+    <h1 id="title">Survey-Form</h1>
+    <p id="description"></p>
+    <form id="survey-form">
+        <label for="name" id="name-label" >Name</label>
+        <input id= "name" type="text" placeholder="Enter your name">
```

Comment:
please Use space-less equal signs.

example <div class="striped"> not <div class = "striped">